### PR TITLE
Add `--pages-branch` parameter

### PR DESCRIPTION
**Pull request by michalbiesek**
Ref:
  helm/chart-releaser@378f527
  helm/chart-releaser@cb0563b

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
need rebase

and maybe we only set that parameter if the user pass it otherwise we keep the existing behavior, wdyt?

**Comment by michalbiesek** [comment]
> need rebase

Rebased.

> and maybe we only set that parameter if the user pass it otherwise we keep the existing behavior, wdyt?

I've changed the implementation. Please take a look.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ A GitHub action to turn a GitHub project into a self-hosted Helm chart repo, usi
 - `skip_existing`: Skip package upload if release/tag already exists
 - `mark_as_latest`: When you set this to `false`, it will mark the created GitHub release not as 'latest'.
 - `packages_with_index`: When you set this to `true`, it will upload chart packages directly into publishing branch.
+- `pages_branch`: The GitHub pages branch
```
Comment (cpanato) [inline README.md:25]:
```suggestion
- `pages_branch`: Name of the branch to be used to push the index and artifacts. (default to: gh-pages but it is not set in the action it is a default value for the chart-releaser binary)
```

**Review by cpanato — CHANGES_REQUESTED** [review]

File: action.yml [hunk 2]
```diff
@@ -35,6 +35,9 @@ inputs:
   packages_with_index:
     description: "Upload chart packages directly into publishing branch"
     required: false
+  pages_branch:
+    description: "The GitHub pages branch"
```
[the remark this answers is not in the document]
Comment (michalbiesek) [reply action.yml]: Done.

**Comment by michalbiesek** [comment]
> need rebase

Rebased.

**Comment by cpanato** [comment]
thanks!

**Pull request merged**